DiamondLightSource/SynchWeb
# [LIMS-1563] Remove touchscreen app

**JIRA ticket**: [LIMS-1563](https://jira.diamond.ac.uk/browse/LIMS-1563)

**Summary**:

To improve security, reduce code size, and ameliorate the repository readability, the touchscreen beamline sample definition page was removed. This page was not in use (given it had been superseded by an internal application), did not work as intended, and was confirmed not to be at use by any beamline.

**Changes**:
- Remove touchscreen app and associated dependencies

**To test**:
- Go to `/touchscreen`, ensure 404 is returned
- Send request to `/visits/{some-visit-id}` ensure 404 is returned
- Check basic SynchWeb functionality as per https://confluence.diamond.ac.uk/display/SSCC/SynchWeb+Tests
